penguin/bravo.c

```diff
@@ -5,7 +5,7 @@
  * to something other than zero.
  */
 
-int penguin_bravo(void) {
+short penguin_bravo(void) {
```

Comment:
I really like this change, using `short` is more appropriate. However, you seem to have forgotten to update the declaration in the `interfaces.h` file as well, and that's why the build fails.

Reply:
I have fixed the interface. Can you please re-check.

Reply:
Looks good, thanks!